Source/configDsp.cmake

```diff
@@ -47,6 +47,11 @@ if (HELIUM OR MVEF OR SUPPORT)
    target_include_directories(${project} PRIVATE "${DSP}/PrivateInclude")
 endif()
 
+if (MVEI OR HELIUM)
```

Comment:
And the case MVEF ? HELIUM is implying MVEI and MVEF but I can imagine a user only using MVEF as a define.

Reply:
The conversion errors (without `-flax-vector-conversions` when using GCC) were in integer vectors, so I didn't add check for `MVEF`. But would
```cmake
if (MVEI OR MVEF OR HELIUM)
```
be better?